src/test/java/io/xpring/xrpl/XrpAccountSetProtoConversionTest.java

```diff
@@ -0,0 +1,44 @@
+package io.xpring.xrpl;
+
+import static org.assertj.core.api.AssertionsForInterfaceTypes.assertThat;
+
+import io.xpring.xrpl.model.XrpAccountSet;
+import org.junit.Test;
+import org.xrpl.rpc.v1.AccountSet;
+
+public class XrpAccountSetProtoConversionTest {
+  @Test
+  public void allFieldsSetTest() {
+    // GIVEN an AccountSet protocol buffer with all fields set.
+    final AccountSet accountSetProto = FakeXrpProtobufs.allFieldsAccountSet;
+    // WHEN the protocol buffer is converted to a native Java object.
+    final XrpAccountSet accountSet = XrpAccountSet.from(accountSetProto);
+
+    // THEN the AccountSet converted as expected.
+    assertThat(accountSet.clearFlag().get()).isEqualTo(accountSetProto.getClearFlag().getValue());
+    assertThat(accountSet.domain().get()).isEqualTo(accountSetProto.getDomain().getValue());
+    assertThat(accountSet.emailHash()).isEqualTo(accountSetProto.getEmailHash().getValue().toByteArray());
+    assertThat(accountSet.messageKey()).isEqualTo(accountSetProto.getMessageKey().getValue().toByteArray());
+    assertThat(accountSet.setFlag().get()).isEqualTo(accountSetProto.getSetFlag().getValue());
+    assertThat(accountSet.tickSize().get()).isEqualTo(accountSetProto.getTickSize().getValue());
+    assertThat(accountSet.transferRate().get()).isEqualTo(accountSetProto.getTransferRate().getValue());
+  }
+
+  @Test
+  public void oneFieldSetTest() {
+    // GIVEN an AccountSet protocol buffer with only one field set.
+    final AccountSet accountSetProto = FakeXrpProtobufs.oneFieldAccountSet;
+
+    // WHEN the protocol buffer is converted to a native Java object.
+    final XrpAccountSet accountSet = XrpAccountSet.from(accountSetProto);
+
+    // THEN the AccountSet converted as expected.
+    assertThat(accountSet.clearFlag().get()).isEqualTo(accountSetProto.getClearFlag().getValue());
```

Comment:
and here

Reply:
per offline, seems like github showed something weird - there's no extra line